src/global/global_init.cc

```diff
@@ -68,6 +78,10 @@ static const char* c_str_or_null(const std::string &str)
 static int chown_path(const std::string &pathname, const uid_t owner, const gid_t group,
 		      const std::string &uid_str, const std::string &gid_str)
 {
+  #ifdef _WIN32
+  return 0;
```

Comment:
with no warning?

Reply:
Right, a log message might be useful. I'd use a low log level such (e.g. 10) so that we don't clutter the logs since this will be the standard behavior on Windows.

Reply:
At a second thought, I see that most of those functions emit warnings and errors directly to stderr because the logging facility might not be initialized at this point, so we can't control the log level.

I'm a bit reluctant to logging a warning to stderr all the time. I've received requests for limiting warning messages for what's considered expected behavior on Windows. Let me know what you think.

Reply:
Reading your reply, I no have no pref.